### DISCUSSION TITLE
SyntaxError: Unexpected token .

### DISCUSSION THREAD

**Discussion by airani** [alpinejs/alpine#3840] [7. Bugs (archived)]
Hi,

After upgrade Alpinejs in some users browser I got this error report:

```
SyntaxError: Unexpected token .
```

https://github.com/alpinejs/alpine/blob/d352364717aa9ee4b12acb7f4d5a5df233933cfb/packages/alpinejs/src/scope.js#L72

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/alpinejs/alpine/discussions/1423

**Comment by airani** [comment]
@ekwoka This problem is caused in your commit, please fix that

**Reply by ekwoka** [reply]
There is nothing to fix there as it is valid code.

If you are running this in/want to support runtimes that do not support [optional chaining](https://caniuse.com/mdn-javascript_operators_optional_chaining), you can setup your bundler to target older specs. Optional chaining is a transpilable feature, so setting your bundler to target es2017 or es2018 will transpile this to a collection of ternaries.

There is little reason for any modern library to not use modern js features when they are transpilable back to the oldest supported target.

If you are using CDN, you can use a cdn that allows target options like esm.sh, or chime in your support for #3625 which targets 2017 for the cdn prebundles.
